README.md

```diff
@@ -1,18 +1,49 @@
 # Faros Events CLI
 CLI for reporting events to Faros platform
 
```

Comment:
I did like your table with all the possible arguments you had. Would you want to put some sort of it here? https://gist.github.com/willmarks/70f1c2151e2aadf9f3caa8078571faf1#required-fields-deployment